src/aerie_cli/__main__.py

```diff
@@ -1,24 +1,28 @@
 """Programmatic entrypoint for CLI application.
 """
 import sys
-from rich.console import Console
-
+import logging
```

Comment:
I know I went back and forth on this, but I decided I like the ergonomics better if we configure a specific child logger in `/src/utils/logger.py`. Then, it's clearer in the code that the logger we import has been specifically configured for this application. For example:

```py
from aerie_cli.utils.logger import aerie_cli_logger as logger

...

logger.info("Whatever")
```